lib/locales/es/frontend/organizations.json

```
@@ -1,7 +1,6 @@
 {
   "all_organizations_title": "Organizaciones que lanzan hackathons y desafíos",
   "all_organizations_page_description": "Startups, empresas, universidades, inversores, comunidades tecnológicas, escuelas de programación y otras organizaciones que lanzan hackathons en la plataforma de hackathons de TAIKAI",
-  "launch_your_challenge": "Lanza tu Hackathon",
```

We are using this on `components/products/launch-challenge-modal.js`
```suggestion
  "launch_your_challenge": "Lanza tu Hackathon",
```